src/helpers/mesh.rs

```diff
@@ -38,6 +40,9 @@ pub trait Mesh {
 
     /// Returns the unique identity of this helper.
     fn identity(&self) -> Identity;
+
+    /// Returns share of value one for this helper.
+    fn share_of_one<F: Field>(&self) -> Replicated<F>;
```

Comment:
This is a state which each helper needs to remember which is controlled by Mesh. mock.rs seems to be the correct place to implement this. Let me know otherwise

Reply:
My thinking is that `share_of_one` implementation is not `Mesh` specific. For example, if we have 3 different implementations of `Mesh` (test,prod,e2e), all of them will implement `share_of_one` in exactly the same way, copy-pasting the same code.


There are two ways to deal with it. First - you can implement `share_of_one` inside `Mesh` trait itself, providing a blanket implementation for everything that implements `Mesh`. Alternatively, you can move it to `Replicated`:

```
   impl Replicated<F> {
         pub fn share_of_one(identity: Identity) -> Self
   }
```

Reply:
I will move it to the trait. That makes more sense to me